docs/roadmap.md

```diff
@@ -0,0 +1,124 @@
+   the end users have with regards to Software Supply Chain and CI/CD and the
+   challenges they are facing and the pain points they are experiencing.
+3. **Shared Vocabulary:** We work on contributing to shared vocabulary from Software
+   Supply Chain perspective.
+4. **Promote Best Practices:** We work on identifying, contributing, and promoting
+   best practices that can help organizations the develop, build, and deliver software
+   in a secure manner.
+
+## RoadMap
+
+![](https://lh5.googleusercontent.com/5F-b4dHm1MMqn47PNzMx9Br7h6N62fEnZU3xp60E9LksEo-MmOK8Ok0PqjyAet4oevSk4vXcgfdi7wg8V4gRuvklCDoO57km-r6DKZS7w4IFhlquyq4_KCHrcYVZNXR1EIRTyDxO)
+
+Reference: Janna Bastow (https://twitter.com/simplybastow/status/1168531737766432768?s=20)
+
+### Now 
+
+#### Knowledge Transfer
+
+The CDF SIG Software Supply Chain holds regular presentations to enable knowledge sharing about
+open source Software Supply Chain tools, technologies, frameworks as well as the techniques used
+by organizations and the initiatives that are undertaken by the CDF and sister communities in order
+to cross-collaborate and pollinate across project, communities, and organizations.
+
+#### End User Requirements
+
+End users will share use cases, challenges, and the work they are doing within
+their organizations with the community in order to enable dialog and further
+collaboration.
+
+In addition to end users presenting to the SIG, [the CDF User Stories Program](https://cd.foundation/blog/2023/02/07/launching-cdf-user-stories-first-up-fidelity-investments/)
+features end user organizations to share their requirements, use cases, challenges, and the initiatives
+they have are in order to share them with the communities on an ongoing basis.
+
+#### Supply Chain Maturity
+
+Supply Chain Maturity metrics are supplemental to the many details [Dora](https://www.devops-research.com/research.html)
+elides and are helpful in identifying gaps in the supply chain that will improve Dora metrics.
+Supply Chain Maturity metrics provide milestones like automated rollout, automated rollback, and deployment
+tracking that provide the tooling and insights needed to minimize recovery time.
+
+The initial version of the [Supply Chain Maturity metrics](https://github.com/cdfoundation/sig-software-supply-chain/blob/main/docs/supply-chain-maturity.md)
+has been developed by the SIG in a collaborative manner and contributions are expected and welcomed in
+order to improve it.
+
+### Next
+
+#### Establishing Shared Vocabulary
+
+Various groups within the CDF such as [SIG Interoperability](https://github.com/cdfoundation/sig-interoperability)
+and [SIG Events](https://github.com/cdfoundation/sig-events) have been working on
+developing a shared vocabulary to help enable the further developments in corresponding
+areas. The work is done within the SIG Interoperability repository and the community is
+encouraged to contriubute to corresponding documents listed below.
```

Comment:
*typo -> contribute 😁

Reply:
Thanks for catching it Batuhan! Updated.